app/queries/scrape_result_search_query.rb

```diff
@@ -10,10 +10,11 @@ class ScrapeResultSearchQuery
     pattern: 'url ~ ?'
   }.freeze
 
-  def initialize(pattern:, query_type:)
+  def initialize(pattern:, query_type:, scope: KeywordSearchEntry)
```

Comment:
I am unsure why this change and the one for `KeywordsQuery` are needed. Please explain.